influxdb_client/client/flux_csv_parser.py

```diff
@@ -45,14 +45,15 @@ class FluxCsvParser(object):
     """Parse to processing response from InfluxDB to FluxStructures or DataFrame."""
 
     def __init__(self, response: HTTPResponse, serialization_mode: FluxSerializationMode,
-                 data_frame_index: List[str] = None, profilers: List[str] = None) -> None:
+                 data_frame_index: List[str] = None, query_options: object = None) -> None:
```

Comment:
Please remove type hint:

```suggestion
                 data_frame_index: List[str] = None, query_options=None) -> None:
```